app/services/hearing_result_fetcher.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
+
+class HearingResultFetcher < ApplicationService
```

Comment:
As we renamed the `HearingResult`model to `Hearing`, would make sense to normalize it into the services too so we have `HearingFetcher` and `HearingRecorder` ?

Reply:
Oops, sorry, did that retrospectively https://github.com/ministryofjustice/laa-court-data-adaptor/pull/4